include/dxc/HLSL/DxilShaderModel.h

```diff
@@ -86,7 +86,7 @@ class ShaderModel {
               unsigned m_NumInputRegs, unsigned m_NumOutputRegs,
               bool m_bUAVs, bool m_bTypedUavs, unsigned m_UAVRegsLim);
 
-  static const unsigned kNumShaderModels = 34;
+  enum { kNumShaderModels = 34 };
```

Comment:
I think `static const` will actually allocate memory for the variable? using anonymous enum will not.

https://stackoverflow.com/questions/8290132/is-memory-allocated-for-a-static-const-variable-whose-address-is-never-used

But for this specific case, maybe the compiler can do the trick to avoid allocation. :)